### DISCUSSION TITLE
Glpi-Agent doesn't provide the last computer boot time

### DISCUSSION THREAD

**Discussion by keytrap-x86** [glpi-project/glpi-agent#294] [Q&A]
With Fusion Inventory I had a saved search which could get me all the computers that haven't been rebooted since xxx.
I can't find this criteria now.
Any idea ?

Thanks

**Comment by g-bougard** [comment]
Hi @kodsu 
you're probably confusing GLPI & GLPI-Agent. GLPI-Agent is generating a XML or a JSON (for GLPI 10) and it should provide this information. You should check the inventory files before telling the agent is not doing the job as you don't find it in GLPI.

**Reply by keytrap-x86** [reply]
Hi, I did this ticket between two other tasks so it eventually had no sense after re-reading it. It's for sure an issue with GLPI and not the agent.
